flink-walkthroughs/flink-walkthrough-common/pom.xml

```diff
@@ -43,4 +43,74 @@ under the License.
 			<scope>provided</scope>
 		</dependency>
 	</dependencies>
+
+	<build>
```

Comment:
What is this plugin used for?

Reply:
Failed to run maven install in flink-walkthrough module because this error
```
Execution default of goal com.github.siom79.japicmp:japicmp-maven-plugin:0.17.1:cmp failed: Could not load 'org.apache.flink.streaming.api.functions.source.FromIteratorFunction': org.apache.flink.streaming.api.functions.source.FromIteratorFunction. Please make sure that all libraries have been added to the classpath 
```
However, adding `org.apache.flink.streaming.api.functions.source.FromIteratorFunction` to exclude in japicmp-maven-plugin of parent pom did not have an effect.

Reply:
Thanks for pointting out this, removed this unnecessary change and fixed it in flink-parent pom.